src/test/java/org/tools4j/hoverraft/server/VoteRequestHandlerTest.java

```diff
@@ -177,13 +186,15 @@ public void onVoteRequest_wrongTerm() throws Exception {
         final VoteRequest voteRequest = DirectMessageFactory.createForWriting()
                 .voteRequest()
                 .term(badTerm)
-                .candidateId(serverId)
-                .lastLogTerm(lastLogTerm)
-                .lastLogIndex(lastLogIndex);
+                .candidateId(serverId);
+        //need LogEntry mutators to return parent object on done()
+        voteRequest.lastLogEntry().term(lastLogTerm)
+                .index(lastLogIndex);
```

Comment:
that's the disadvantage of the nested object structure --- the method chaining breaks down :-(